module-4/backend/week_two.md

```diff
@@ -0,0 +1,32 @@
+## Week Two - Module 4 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+
+Note: When you're done, submit a PR. 
+
+1. What is Webpack and why is it useful?
+A frontend frame work, you can then update the application with live changes
+2. When do you want to use event delegation?
+When an events functionality is takes longer than normal
+3. What's one difference between ES5 and ES6?
+semi colons are handled for you in ES6
+4. How are you using the MVC design pattern in your Quantified Self project?
+We have different folders for each one. similar to the way ruby does it
```

Comment:
ruby doesn't have an MVC pattern by default. rails does follow the MVC design pattern though